apis/rds/v1alpha1/custom_types.go

```diff
@@ -192,9 +192,15 @@ type CustomDBClusterParameters struct {
 
 	// The password for the master database user. This password can contain any
 	// printable ASCII character except "/", """, or "@".
+        // 
+        // This parameter will be required in the following scenarios:
+        // - The first cluster for a global Aurora cluster
+        // - Any cluster as long as it doesn't belong to a global Aurora cluster
+        //
+        // It won't be required when trying to attach a secondary regional cluster to an existing global cluster.
 	//
-	// Constraints: Must contain from 8 to 41 characters. Required.
-	MasterUserPasswordSecretRef *xpv1.SecretKeySelector `json:"masterUserPasswordSecretRef"`
+	// Constraints: Must contain from 8 to 41 characters.
+	MasterUserPasswordSecretRef *xpv1.SecretKeySelector `json:"masterUserPasswordSecretRef,omitempty"`
```

Comment:
Thanks for the change. However, you have to run `make generate` and commit the changes in `package`. For more info see the [contribution guidelines](https://github.com/crossplane-contrib/provider-aws/blob/master/CONTRIBUTING.md).

 Additionally, it would be great if you could provide an example test case.

Reply:
How do you mean by `example test case`. Is it just providing a new example as the ones in the `examples` directory here, https://github.com/crossplane-contrib/provider-aws/tree/master/examples? If so, I'm more than happy to create a follow-up request with one 💪

Reply:
Yes, please provide one (or more if necessary) sample MRs in `examples` with this PR.